.gitlab-ci.yml

```diff
@@ -389,3 +400,47 @@ cancel-pipeline-check-runtime-migration-westend:
   extends: .cancel-pipeline-template
   needs:
     - job: check-runtime-migration-westend
+
+# Asset hub chains
+cancel-pipeline-check-runtime-migration-asset-hub-polkadot:
+  extends: .cancel-pipeline-template
+  needs:
+    - job: check-runtime-migration-asset-hub-polkadot
+
+cancel-pipeline-check-runtime-migration-asset-hub-kusama:
+  extends: .cancel-pipeline-template
+  needs:
+    - job: check-runtime-migration-asset-hub-kusama
+
+cancel-pipeline-check-runtime-migration-asset-hub-westend:
+  extends: .cancel-pipeline-template
+  needs:
+    - job: check-runtime-migration-asset-hub-westend
+
+# Bridge hub
+cancel-pipeline-check-runtime-migration-bridge-hub-polkadot:
+  extends: .cancel-pipeline-template
+  needs:
+    - job: check-runtime-migration-bridge-hub-polkadot
+
+cancel-pipeline-check-runtime-migration-bridge-hub-kusama:
+  extends: .cancel-pipeline-template
+  needs:
+    - job: check-runtime-migration-bridge-hub-kusama
+
+cancel-pipeline-check-runtime-migration-bridge-hub-rococo:
+  extends: .cancel-pipeline-template
+  needs:
+    - job: check-runtime-migration-bridge-hub-rococo
+
+# Collectives
+cancel-pipeline-check-runtime-migration-collectives-polkadot:
+  extends: .cancel-pipeline-template
+  needs:
+    - job: check-runtime-migration-collectives-polkadot
+
+# Contracts
+cancel-pipeline-check-runtime-migration-contracts-rococo:
+  extends: .cancel-pipeline-template
+  needs:
+    - job: check-runtime-migration-contracts-rococo
```

Comment:
This part is needed only for jobs that are marked as `Required` in GitHub. Currently these jobs are not mandatory to pass so this part should be either removed, or commented if we want to make them `Required`

Reply:
@ggwpez do you know why only rococo and westend were set to required? Thoughts on making all production chains required?

Reply:
Since we move the production chains out, we can mark them all as non-required here (Rococo, Westend + Parachains)